src/Core/Utilities/StaticStore.cs

```diff
@@ -151,7 +148,7 @@ static StaticStore()
 
     /// <summary>
     /// Determines if the stripe plan id is an addon item by checking if the provided stripe plan id
-    /// matches either the <see cref="Plan.StripeStoragePlanId"/> or <see cref="Plan.StripeServiceAccountPlanId"/>
+    /// matches either the <see cref="Plan.PasswordManager.StripeStoragePlanId"/> or <see cref="Plan.SecretsManager.StripeServiceAccountPlanId"/>
```

Comment:
You still have a lot of warnings here that the XMLdoc references couldn't be resolved.

I found that you have to refer to it like this, I don't know why:  `Plan.PasswordManagerPlanFeatures.StripeStoragePlanId` and `Plan.SecretsManagerPlanFeatures.StripeServiceAccountPlanId`